src/rollbar.ts

```diff
@@ -84,7 +84,7 @@ class RollbarController {
      */
     error(what: string, additional: object = {}): void {
         log.error(what, JSON.stringify(additional));
-        debugger;
+        // debugger;
```

Comment:
Why this change?

Reply:
oh whoops sorry I was supposed to revert that. It was for testing. Will fix

Reply:
shoudl be fixed